package.json

```diff
@@ -5,7 +5,9 @@
   "main": "index.js",
   "scripts": {
     "test": "nyc mocha --require @babel/register ./tests/*.js  --timeout 25000 --exit",
-    "coverage": "nyc report --reporter=text-lcov | coveralls"
+    "coverage": "nyc report --reporter=text-lcov | coveralls",
+    "start": "set NODE_ENV=dev&& nodemon ./src/index.js --exec babel-node --presets @babel/preset-env",
+    "db:connect": "set NODE_ENV=dev&& babel-node ./src/database/models/ connect"
```

Comment:
No need to set the node_env here. Lets make the env depend on what is set in the .env file. @karamuka please check this as it is related to your .env PR.